www/js/footer-left.json

```diff
@@ -1,14 +1,37 @@
 {
   "search": {
     "label": "Search",
-    "icon": "search"
+    "icon": "search",
+    "type": "pane-nav"
   },
-  "session": {
+  "history": {
     "label": "History",
-    "icon": "history"
+    "icon": "history",
+    "type": "tab-nav"
   },
   "shabad": {
     "label": "Current Shabad",
-    "icon": "dot-circle-o"
+    "icon": "dot-circle-o",
+    "type": "pane-nav"
+  },
+  "themes": {
+    "label": "Themes",
+    "icon": "paint-brush",
+    "type": "tab-nav"
+  },
+  "insert": {
+    "label": "Insert",
+    "icon": "desktop",
+    "type": "tab-nav"
+  },
+  "display": {
+    "label": "Display",
```

Comment:
Should it still be called Display? its more like settings or preferences now since it has all the other options. @maneetpaul

Reply:
I agree that this should be changed. I think "Settings" would be more appropriate.

Reply:
Okay.